examples/unsloth/unsloth.yaml

```
@@ -0,0 +1,26 @@
+# Runs the unsloth example app via SkyPilot
+#
+# The example app starts by obtaining the LAION dataset,
+# loads the Mistral model with 4-bit precision, performs model
+# patching with fast LoRA weights, and finally initializes and
+# trains the model using the SFTTrainer with specified
+# hyperparameters and the LAION dataset.
+#
+# Usage:
+#   sky launch -c myclus unsloth.yaml
+
+resources:
+  accelerators: T4:1
```

If the script works on different GPUs, would be nice to showcase how skypilot can optimize between multiple accelerator types:
```suggestion
  accelerators: {'T4:1', 'L4:1', 'A100:1'}
```

---

Unfortunately some of the `pip` installation lines change for unsloth depending on the GPUs used (Ampere vs Turing vs other architectures) so probably going to have to hold off here? But open if you have other ideas/workarounds!